pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.styles.ts

```diff
@@ -0,0 +1,174 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { selectThemeVariant, stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const backgroundColor = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#161719' }, theme.type);
+  // @ts-ignore
```

Comment:
Let's get rid of `ts-ignore` by casting to any. A tempo solution, I'm sure upstream will fix it soon.